spec/01-unit/19-hybrid/04-wrpc_spec.lua

```diff
@@ -0,0 +1,273 @@
+local helpers = require "spec.helpers"
```

Comment:
this is probably still too blackbox-ish for a unit test, let's move it to integration test folder. for unit test, it would be useful to test following items:
- proto.lua: parse annotations; a RPC message be succefully handled only after set_handler is called
- queue.lua, threads.lua, future.lua: functional test for exposed functions